ETL/Pipeline/test_extract.py

```diff
@@ -0,0 +1,98 @@
+"""This file tests whether the extract file works as expected"""
+from unittest.mock import patch
+import pytest
+from extract import (get_product_info, get_url, get_current_price,
+                     get_sale_status, process_product, populate)
```

Comment:
```suggestion
"""This file tests whether the extract file works as expected"""

from unittest.mock import patch
import pytest

from extract import (get_product_info, get_url, get_current_price,
                     get_sale_status, process_product, populate)
```
Spaces look nicer and separates the third party libraries from your files

Reply:
Added and noted for future projects.